eZ/Publish/API/Repository/Values/User/RoleUpdateStruct.php

```diff
@@ -23,4 +23,11 @@ class RoleUpdateStruct extends ValueObject
      * @var string
      */
     public $identifier;
+
+    /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT
+     */
+    public $status;
```

Comment:
This should not be here as drafts are created through `RoleService::createRoleDraft(Role $role)`.
But maybe I'm missing something?